azure-kusto-data/azure/kusto/data/kcsb.py

```diff
@@ -128,12 +128,13 @@ def __init__(self, connection_string: str):
         for kvp_string in connection_string.split(";"):
             key, _, value = kvp_string.partition("=")
             keyword = self.ValidKeywords.parse(key)
+            value_stripped = value.strip()
             if keyword.is_str_type():
-                self[keyword] = value.strip()
-            if keyword.is_bool_type():
-                if value.strip() in ["True", "true"]:
+                self[keyword] = value_stripped[:-1] if value_stripped[-1] == "/" else value_stripped
```

Comment:
This sort of voodoo blood magic requires a comment about what's the expected structure of the string being parsed.
;)

Reply:
Is this trimming a trailing slash?
if so, you can use .rstrip("/")